data/json/scenarios.json

```
@@ -824,5 +824,16 @@
     "start_name": "Retirement Community",
     "professions": [ "senior", "old_vet" ],
     "flags": [ "CITY_START", "LONE_START" ]
+  },
+  {
+    "type": "scenario",
+    "id": "nuclear_plant_start",
+    "name": "Nuclear power plant",
+    "points": 2,
```

```suggestion
    "points": 1,
```
I wouldn't take my word for this. Perhaps wait for more feedback from players to balance it. Anyway, everyone will just say "play freeform", "balance is subjective", "skill issue", blah blah blah...

A skilled player can make it worth two points for sure, especially if they try clearing it - I mean, there's some interesting loot here: electronics, cars, axes and armor if you know where to look - but atm with some of the terrifying monsters lurking around, the lack of medication/first aid, and the lack of exits, this is actually a pretty tough map to escape. I died once - quickly - and made it out twice in about 30 minutes, although pretty battered.